pkg/master/master.go

```diff
@@ -65,17 +65,21 @@ import (
 
 // Config is a structure used to configure a Master.
 type Config struct {
-	Client                 *client.Client
-	Cloud                  cloudprovider.Interface
-	EtcdHelper             tools.EtcdHelper
-	EventTTL               time.Duration
-	MinionRegexp           string
-	KubeletClient          client.KubeletClient
-	PortalNet              *net.IPNet
-	EnableLogsSupport      bool
-	EnableUISupport        bool
-	EnableSwaggerSupport   bool
-	EnableV1Beta3          bool
+	Client            *client.Client
+	Cloud             cloudprovider.Interface
+	EtcdHelper        tools.EtcdHelper
+	EventTTL          time.Duration
+	MinionRegexp      string
+	KubeletClient     client.KubeletClient
+	PortalNet         *net.IPNet
+	EnableLogsSupport bool
+	EnableUISupport   bool
+	// allow downstream consumers to disable swagger
+	EnableSwaggerSupport bool
+	// allow v1beta3 to be conditionally enabled
+	EnableV1Beta3 bool
```

Comment:
Yeah, comments reset the margin on the right items.

----- Original Message -----
> > -	EnableUISupport        bool
> > -	EnableSwaggerSupport   bool
> > -	EnableV1Beta3          bool
> > +	Client            *client.Client
> > +	Cloud             cloudprovider.Interface
> > +	EtcdHelper        tools.EtcdHelper
> > +	EventTTL          time.Duration
> > +	MinionRegexp      string
> > +	KubeletClient     client.KubeletClient
> > +	PortalNet         *net.IPNet
> > +	EnableLogsSupport bool
> > +	EnableUISupport   bool
> > +	// allow downstream consumers to disable swagger
> > +	EnableSwaggerSupport bool
> > +	// allow v1beta3 to be conditionally enabled
> > +	EnableV1Beta3 bool
> 
> Did gofmt do this?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/3997/files#r23884082